cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -38,18 +42,26 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.*;
+import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import static java.util.function.Predicate.not;
+
 /**
  * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
  */
 @AutoService(Importer.class)
 public class CgmesImport implements Importer {
 
-    enum FictitiousSwitchesCreationMode {
+    public enum FictitiousSwitchesCreationMode {
         ALWAYS,
         ALWAYS_EXCEPT_SWITCHES,
-        NEVER;
+        NEVER
+    }
+
+    public enum AssembledSeparatingBy {
```

Comment:
As I said in the previous PR, "assembled" is a name choice hard to understand for the community and for non CGMES expert. In Powsybl, assembled is a network. Here the question is just : do we want to add subnetworks in a network and how do we define a subnetwork. `NAME` is too global, maybe something like `FILE_NAME` (and you can keep `MODELING_AUTHORITY`).